First branch testing

holywyvern opened the pull request:
## Summary 

- Made some changes to test branches
- Testing pull requests
- Checking if markdown works on my comment.

## Trello Card
https://trello.com/c/QqO7DiBm/37-tutorial-git-github

SKOLZ commented on app/assets/stylesheets/commons.scss:
```diff
@@ -0,0 +1,134 @@
+  .phone::after {
+    content: ' ';
+    position: absolute;
+    background: linear-gradient(180deg, white, $orange, white);
```
use color variables for the white value

SKOLZ commented on app/assets/stylesheets/commons.scss:
```diff
@@ -0,0 +1,134 @@
+      padding: 10px 0;
+      line-height: 25px;
+      font-size: 19px;
+      color: white;
```
color variable

SKOLZ commented on app/assets/stylesheets/commons.scss:
```diff
@@ -0,0 +1,134 @@
+  .feature-title {
+    font-size: 18px;
+    font-weight: bold;
+    color: #555;
```
color variable

SKOLZ commented on app/assets/stylesheets/commons.scss:
```diff
@@ -0,0 +1,134 @@
+  }
+
+  .btn-subscribe {
+    color: white;
```
color variable

SKOLZ commented on app/assets/stylesheets/commons.scss:
```diff
@@ -0,0 +1,134 @@
+  .phone {
+    display: none;
+  }
+}
```
leave an enter at the end of each file

SKOLZ commented on app/assets/stylesheets/variables/colors.scss:
```diff
@@ -0,0 +1,4 @@
+$orange: #f5c85f;
+$gray-bg: #EEEEEE;
+$dark-gray-text: #111111;
+$gray-text: #999999;
```
enter at the end of the file

SKOLZ replied:
```
#999

SKOLZ commented on app/assets/stylesheets/variables/colors.scss:
```diff
@@ -0,0 +1,4 @@
+$orange: #f5c85f;
+$gray-bg: #EEEEEE;
```
you should write that colour like this:
```
#EEE

SKOLZ commented on app/assets/stylesheets/variables/colors.scss:
```diff
@@ -0,0 +1,4 @@
+$orange: #f5c85f;
+$gray-bg: #EEEEEE;
+$dark-gray-text: #111111;
```
```
#111

SKOLZ commented on app/views/application/maquetado.html.slim:
```diff
@@ -0,0 +1,33 @@
+    .more-info
+      div.info-updates
+        | Get info on new updates
+      div
```
mmmmmm this is an unnecesary div, remove this one and add style to the input so it works the way you want.

SKOLZ commented on app/views/application/maquetado.html.slim:
```diff
@@ -0,0 +1,33 @@
+        | Get info on new updates
+      div
+        input.suscription-input
+      div
```
same here

SKOLZ commented on app/views/application/maquetado.html.slim:
```diff
@@ -0,0 +1,33 @@
+        p
+          | All your debts are kept safe in your phone.
+    .more-info
+      div.info-updates
```
div is the default element so you could avoid writing it. It looks like this:
```
.info-upadates
```
but in this particular case as you are wrapping a text this shouldn't be a div. this should be a headline for example.

SKOLZ commented:
looks good :) :+1:

Commit:
# First branch testing

- Made some changes to test branches
- Testing pull requests
- Checking if markdown works on my comment.

leamotta reviewed (changes requested):
You can close this PR.
Also, check your git configuration so that your commits appear with your username :)
